cpg_infra/driver.py

```diff
@@ -173,6 +172,10 @@ def __init__(
         ] = defaultdict()
 
     def resolve_dataset_order(self):
+        """
+        This is now obsolete because we setup all datasets, and then deploy
+        so they all exist before the actual deploy.
+        """
```

Comment:
If it's obselete, can we remove it? :)

Reply:
I think it influences the pulumi create order, which is useful when spinning up things - but it doesn't really matter. I've left it because it's a useful code snippet to have around.